FlowCrypt/src/androidTest/java/com/flowcrypt/email/ui/base/BaseMessageDetailsFlowTest.kt

```diff
@@ -198,11 +200,24 @@ abstract class BaseMessageDetailsFlowTest : BaseTest() {
       .check(matches(withText(Formatter.formatFileSize(getContext(), att.encodedSize))))
   }
 
-  protected fun checkWebViewText(text: String?) {
+  protected fun checkWebViewText(html: String?) {
+    val document = TagSoupDocumentParser.newInstance().parse(html)
+    val bodyElements = document.getElementsByTagName("body")
+
+    val elementList = mutableListOf<Node>()
+    for (position in 0 until bodyElements.length) {
+      elementList.add(bodyElements.item(position))
+    }
+    val content = elementList.joinToString(separator = "") {
+      if (it.textContent.isNotEmpty()) {
+        it.textContent + "\n"
+      } else ""
+    }.trim()
```

Comment:
maybe we can simplify getting content like below
```kt
val content = (0 until bodyElements.length)
      .asSequence()
      .map { bodyElements.item(it) }
      .filter { it.textContent.isNotEmpty() }
      .joinToString(separator = "\n") { it.textContent }
      .trim()
```

Reply:
perfect!